wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -125,6 +124,128 @@ def _check_modulo_patterns(
             self.add_violation(consistency.ModuloStringFormatViolation(node))
 
 
+@final
+class WrongFormatStringVisitor(base.BaseNodeVisitor):
+    """Restricts usage of ``f`` strings."""
+
+    _valid_format_index: ClassVar[AnyNodes] = (
+        ast.Str,
+        ast.Num,
+        ast.Name,
```

Comment:
What about `False`, `None`, and `True`?

Reply:
You mean as indexes? Such as `first[True]`? Is this to be allowed?

Reply:
I guess so